backend/corpora/lambdas/api/v1/authentication.py

```diff
@@ -75,9 +75,15 @@ def oauth2_callback() -> Response:
     """API call: redirect from the auth server after login successful."""
     config = CorporaAuthConfig()
     client = get_oauth_client(config)
-    token = client.authorize_access_token()
-    # write the cookie
-    save_token(config.cookie_name, token)
+    try:
+        token = client.authorize_access_token()
+        # write the cookie
+        save_token(config.cookie_name, token)
+    except Exception as e:
+        current_app.logger.warning(f"Unable to authorize access token: {str(e)}: {str(token)}")
```

Comment:
do we want to continue raising the expectation so we return an error response.

Reply:
We can raise an UnauthorizedError here.  I agree that is better.